modules/standard/IO.chpl

```diff
@@ -2494,7 +2512,7 @@ proc channel._set_style(style:iostyle) {
 /*
 
    Return the locale on which an ongoing I/O was started with a channel.
-   This method will return nil unless it is called on a channel that is
+   This method will return ``nil`` unless it is called on a channel that is
```

Comment:
It's not actually ``nil`` because locales are now records. Looks like it uses `nilLocale` now. However `nilLocale` is not documented.

Reply:
I changed it to `nilLocale` even if that's not documented elsewhere, for accuracy.